brainiak/funcalign/dl.py

```diff
@@ -0,0 +1,581 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Multi-subject Dictionary Learning
+
+The implementation is based on the following publication:
+
+.. [Varoquaux2011] "Multi-subject Dictionary Learning to Segment an Atlas of
+   Brain Spontaneous Activity",
+   G. Varoquaux, A. Gramfort, F. Pedregosa, V. Michel, .B. Thirion
+   22nd International Conference Information Processing in Medical Imaging
+   (IPMI 2011)
+   http://rd.springer.com/chapter/10.1007/978-3-642-22092-0_46
+
+"""
```

Comment:
Please add a paragraph stating this code implements a different algorithm from nilearn.